guides/dev/issues_readme/README.md

```diff
@@ -17,7 +17,9 @@ Most of the issues are reproducible on the demo. If you have a different case th
 
 ## Avoid Duplicates
 
-Before you file a new ticket, make sure your issue has not been reported yet. Use the [search page](https://dev.ckeditor.com/search) to check it.
+Before you file a new ticket, make sure your issue has not been reported yet. Use the [search page](https://github.com/ckeditor/ckeditor-dev/issues) to check it.
+If the issue is already reported just put a (+1) reaction in it. During our transition period you could also check [CKEditor's former bug tracker](https://dev.ckeditor.com). If the issue
```

Comment:
Put a real 👍 emoji, and leave +1 in the parentheses. Most of OS will render the emoji.